Browser_IDE/stylesheet.css

```diff
@@ -455,22 +449,22 @@ textarea {
 .node-button{
     background: none;
     border: none;
-    color: white;
+    color: var(--primary);
     padding: 0;
     margin: 0;
 }
 .node-button:hover{
-    color: lime;
+    color: var(--iconHover);
 }
 .node-transient{
     opacity: 0.7;
-    color: #f8f8f2;
+    color: var(--errorColour);
```

Comment:
This PR looks pretty good!

Just had one comment on this part - the transient files aren't errors per-se, they're just files that don't exist as part of the saved project (temporary log files, etc).

They'll pretty much always exist when running the projects so maybe a different colour might be clearer to the user?